odoo/addons/base/models/ir_http.py

```diff
@@ -420,14 +420,17 @@ def _binary_record_content(
             if extension:
                 filename = "%s%s" % (filename, extension)
 
-        if not filehash:
+        if not filehash and not xaccel_header:
             filehash = '"%s"' % hashlib.md5(str(base64.b64encode(content)).encode('utf-8')).hexdigest()
 
         status = 200 if content else 404
         return status, content, filename, mimetype, filehash
 
-    def _binary_set_headers(self, status, filename, mimetype, unique, filehash=None, download=False):
+    def _binary_set_headers(self, status, filename, mimetype, unique, filehash=None, download=False, xaccel_header=False):
         headers = [('Content-Type', mimetype), ('X-Content-Type-Options', 'nosniff'), ('Content-Security-Policy', "default-src 'none'")]
+        if xaccel_header:
+            headers.append(xaccel_header)
+            headers.append(('X-Accel-Buffering', "yes"))
```

Comment:
Why are you hardcoding this to its default value?

Reply:
Oh, my bad I thought the default was no for some reason. Will remove this then, it's useless.